build/README.md

```diff
@@ -175,15 +179,17 @@ to be open to UDP traffic.
 
 First step is to create a Google Cloud Project at https://console.cloud.google.com or reuse an existing one.
 
-The build tools (by default) maintain configuration for gcloud within the `build` folder, so as to keep
+The build tools (by default) maintain configuration for gcloud within the `build` folder, to keep
 everything separate (see below for overwriting these config locations). Therefore, once the project has been created,
 we will need to authenticate out gcloud tooling against it. To do that run `make gcloud-init` and fill in the
 prompts as directed.
 
 Once authenticated, to create the test cluster, run `make gcloud-test-cluster`, which will use the Terraform
 configuration found in the `build/terraform/gke` directory.
 
-You can customize GKE cluster via environment variables or by using a [`local-includes`](./local-includes) file.
+You can customize GKE cluster via appending them to your amke target, via environment variables, or by setting them 
```

Comment:
"customize the GKE cluster by appending the following parameters to your make target, ..."